lua/nvim-lsp-installer/servers/omnisharp/init.lua

```diff
@@ -17,38 +18,40 @@ return function(name, root_dir)
             context.use_github_release_file(
```

Comment:
Seeing as the server no longer is a standalone executable and instead runs via the dotnet SDK, I think it'd be nice to make sure it's available on the system (although it's not necessary to just install it):

```suggestion
            std.ensure_executables {
                {
                    "dotnet",
                    "dotnet was not found in path. Refer to https://dotnet.microsoft.com/download for installation instructions.",
                },
            },
            context.use_github_release_file(
```

Reply:
Thanks ^^